Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -911,15 +942,14 @@ def network_stop(self, removestate, removecontent):
             pstate = self.network_get_persistent_state()
             if self.handle is not None:
                 self._logger.debug("LibtorrentDownloadImpl: network_stop: engineresumedata from torrent handle")
+                self.pstate_for_restart = pstate
```

Comment:
same here, if `on_save_resume_data()` can fetch it. Why are we doing the same in two other places?